client/my-sites/customer-home/main.jsx

```diff
@@ -77,7 +77,7 @@ const Home = ( {
 				<FormattedHeader
 					headerText={ translate( 'My Home' ) }
 					subHeaderText={ translate(
-						'Your home base for all the posting, editing, and growing of your site'
+						'Your home base for posting, editing, and growing your site.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.